sim/display.py

```diff
@@ -85,15 +86,34 @@ def get_state_mtx(self):
 
         return np.array(state), names
 
-    def restock(self):
-        """
-        Max Capcity is all available slots x max_per_slot
+    def restock(self, product_with_counts_dict):
+
+
+        # Input
+        # product_with_counts_dict = {"coca_cola": 5, "sprite": 1}
+        # n_slots = 6
+        # max_per_slot = 4
+
+        updated_state = {}
```

Comment:
We need to mutate the internal dictionary of `InventoryProducts` by accessing `self.inv`. This implementation doesn't actually change the levels of the products... it only copies to another dictionary